### PR TITLE
New stamp for NRI

### PR DESCRIPTION
**Pull request by VastKilleroOm**
## About The Pull Request

Add a new stamp for NRI docks and paperworks

Stamp with text "Флотилия Её Величества". In translation "Her Majesty's Fleet"

P.s The letters are made so on purpose in order to show the shabby seal

## How This Contributes To The Skyrat Roleplay Experience

After that, there will be no more "interesting" moments when the same NRI pirates "Diplomats" playing RP will not be able to show the station command documents about the "debt"

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
Screenshot_1237
Sorry. I forgot add stamp icon looks like in game.
image

</details>

## Changelog

:cl: Vlada Frolova
image: added icons for stamp andicon for stamp sign
/:cl:

### CODE REVIEW

**Comment by VastKilleroOm** [comment]
10 failing, 10 successful, and 2 skipped checks
Good work. I think

File: modular_skyrat/master_files/code/modules/paperwork/stamps.dm [hunk 1]
```diff
@@ -67,3 +67,10 @@
 
 		else
 			return
+
+/obj/item/stamp/nri
+	name = "Novaya Rossiyskaya Imperia Stamp"
```
Comment (Zonespace27) [inline modular_skyrat/master_files/code/modules/paperwork/stamps.dm]:
```suggestion
	name = "/improper Novaya Rossiyskaya Imperia stamp"
```

File: code/modules/asset_cache/assets/paper.dm [hunk 2]
```diff
@@ -13,6 +13,10 @@
 		"stamp-paw_green" = 'modular_skyrat/master_files/icons/stamp_icons/paw_green.png',
 		// SKYRAT ADDITION: END - Donator stamp icons
 
+		// SKYRAT ADDITION: START - Other stamp icons
+		"stamp-nri" = "modular_skyrat/master_files/icons/stamp_icons/large_stamp-nri.png",
```
Comment (Zonespace27) [inline code/modules/asset_cache/assets/paper.dm]:
```suggestion
		"stamp-nri" = 'modular_skyrat/master_files/icons/stamp_icons/large_stamp-nri.png',
```

**Review by Zonespace27 — DISMISSED** [review]
Review does not touch sprites/images

**Review by desertmote — APPROVED** [review]

**Pull request merged**